system_helpers/update_alternatives/script.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import logging
+import pathlib
+
+import typeguard
+
+from system_helpers.update_alternatives.alternatives import update_alternatives
+
+@typeguard.typechecked
+def parse_args() -> argparse.Namespace:
+    """
+    Parse CLI arguments.
+    """
+    parser = argparse.ArgumentParser()
+
+    subparsers = parser.add_subparsers(required = True)
+
+    parser_ua = subparsers.add_parser('update-alternatives')
```

Comment:
We should not use a sub-parser, because it's now a standalone script for `update-alternatives`.
Usage should be:
```bash
update-alternative-helpers <args>
```
instead of current:
```bash
update-alternatives-helpers update-alternatives <args>
```

Reply:
Indeed. It's probably the case for the "install packages" too.

Reply:
Well, for `apt-helpers` I would still keep the `install` subparser, because `apt` has many other actions possible.

Reply:
OK. I agree!